docs/source/changelog.rst

```diff
@@ -23,6 +23,9 @@ Added
    - url ``/messages/state/message_id`` return the state of a given ``message_id``
 * Class |TestMessageStateAPI| responsible to test the API of |StateBackend|
 * Add |Flask| as an extra dependency
+* Add |CurrentMessage| Middleware that exposes the current message via a thread local variable, useful to access the message within the actor
+* Add |TestCurrentMessage| responsible to test the new middleware
```

Comment:
you don't need to talk abou tests in changelogs, they are for library users. They just want to know the fonctionalities